pkg/operator/deploy/deploy.go

```diff
@@ -160,12 +162,19 @@ func (o *operator) createDeploymentData(ctx context.Context) (deploymentData, er
 		return deploymentData{}, err
 	}
 
-	return deploymentData{
+	data := deploymentData{
 		IsLocalDevelopment:           o.env.IsLocalDevelopmentMode(),
 		Image:                        image,
 		SupportsPodSecurityAdmission: usePodSecurityAdmission,
 		Version:                      version,
-	}, nil
+	}
+
+	if o.oc.UsesWorkloadIdentity() {
+		data.UsesWorkloadIdentity = o.oc.UsesWorkloadIdentity()
+		data.TokenVolumeMountPath = pkgoperator.OperatorTokenFile
```

Comment:
If we want to place the token at `OperatorTokenFile` (= `/var/run/secrets/openshift/serviceaccount/token`), the mount path should be the directory of `OperatorTokenFile`.

Reply:
Thanks for catching that :facepalm: 

Fixed